cypress/integration/context-selection/categories/index.js

```diff
@@ -106,6 +106,23 @@ Given(
     }
 )
 
+Given(
+    'a data set, org unit, period and some but not all options have been selected and there are some categories connected to the data set',
+    () => {
+        cy.visit(
+            '/#/?dataSetId=lyLU2wR22tC&orgUnitId=ImspTQPwCqd&periodId=202212&attributeOptionComboSelection=LFsZ8v5v7rq-CW81uF03hvV'
+        )
+        cy.get('[data-test="data-set-selector"]').should('exist')
+    }
+)
+
```

Comment:
Duplicate of the same step definition right above this one

Reply:
fixed. sorry